### ISSUE TITLE
The string "Some examples of what this program may provide:" isn't applicable for all the benefits 

### ISSUE THREAD

**Issue by courtneycl** [veteransaffairscanada/vac-benefits-directory#1651]
The see more content in some of our benefits show examples, but some show extra information. We should have different strings for the different cases/ensure that we're writing content for a consistent case (I spot at least three cases: examples, relevant information, a mix of examples and relevant information). 

eg: 
relevant info
![image](https://user-images.githubusercontent.com/3474250/50603863-0b566d80-0e8a-11e9-9b45-556f568e2b6f.png)

relevant info and examples 
![image](https://user-images.githubusercontent.com/3474250/50604115-e95dea80-0e8b-11e9-8fe4-9243621b45b7.png)

What I think might help: 
* identify all the different benefit content cases
* identify what the string should be for each of those cases 
* refer to those cases when writing and validating see more content to ensure they're matched properly and the end product makes sense 

OR
* change the existing string to be less prescriptive

Open to ideas and any thoughts people have had on this. 

cc @arothmel, @scottlester1

**Comment by scottlester1** [comment]
Yes, definitely. The see more content was rushed and needs attention. I've started going through the see more content on our excel spreadsheet. I've also added the column to the word document that program experts were sent. Most are ready to start contributing to this section of the card now.

Having "Some examples of what this program may provide:" at the beginning works for quite a few benefits, but it doesn't work for all of them. If it doesn't make sense to say what the program or benefit provides, could we add a tip or recommendation instead? For example: 

Family Peer Support (OSISS)
Peer support for the families of Veterans living with mental health issues.

See more: 
Tip: Your first step is to contact a Family Peer Support Coordinator. Call 1-800-883-6094 or find a coordinator <link> in your area.

Could we have a quick phone call tomorrow to discuss what you think would help? I'm not sure I follow 100%. Would like to make sure we're on the same page :)
